javapeople.yaml

```diff
@@ -385,6 +385,10 @@
   twitter: "yawkat"
   fediverse: "yawkat@infosec.exchange"
   github: "yawkat"
+- name: "Jonas TM"
+  twitter: "jonas_tm"
```

Comment:
Is this your twitter handle? https://twitter.com/jonas_tm

Reply:
Thanks. It was wrong, updated my twitter handle.